src/client.rs

```diff
@@ -81,32 +81,30 @@ impl MainState {
                     self.game_state = state
                 },
                 ServerMessage::PlaySound(sound, pos) => {
+                    fn play_sound(soundeffect: &sdl2::mixer::Chunk) {
+                        if let Err(e) = sdl2::mixer::Channel::all().play(
+                            soundeffect, 0
+                        ) {
```

Comment:
Weird indentation, but I guess that's fine

Reply:
No you're wrong :P

Reply:
:(